dotnet/targets/Xamarin.Shared.Sdk.DefaultItems.targets

```
@@ -23,6 +23,19 @@
     <None Include="@(ObjcBindingCoreSource)" />
   </ItemGroup>
 
+  <PropertyGroup Condition="'$(_PlatformName)' == 'MacCatalyst'">
+    <EnableDefaultMacCatalystReleaseEntitlements Condition="'$(EnableDefaultMacCatalystReleaseEntitlements)' == ''">True</EnableDefaultMacCatalystReleaseEntitlements>
+    <EnableDefaultMacCatalystDebugEntitlements Condition="'$(EnableDefaultMacCatalystDebugEntitlements)' == ''">True</EnableDefaultMacCatalystDebugEntitlements>
+  </PropertyGroup>
+
+  <ItemGroup Condition="'$(EnableDefaultMacCatalystDebugEntitlements)' == 'True' and '$(Configuration)' == 'Debug'">
+    <CustomEntitlements Include="com.apple.security.get-task-allow" Type="boolean" Value="true" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(EnableDefaultMacCatalystReleaseEntitlements)' == 'True' and '$(Configuration)' == 'Release'">
+    <CustomEntitlements Include="com.apple.security.app-sandbox" Type="boolean" Value="true" /
```

Looks like you're missing a character here:

```suggestion
    <CustomEntitlements Include="com.apple.security.app-sandbox" Type="boolean" Value="true" />
```